lib/common/template.js

```diff
@@ -41,7 +41,7 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
             Constants.Templates.Directory
         ).then(function (templates) {
             var promises = _.map(templates, function (contents, name) {
-                return self.put(name, contents);
+                return self.put(name, contents.toString());
```

Comment:
Yeah, the fs library will give us an empty buffer for an empty file. If the file does not exist, we'll have failed before we get to this point:

```
> e = fs.readFileSync('./nothere')
Error: ENOENT, no such file or directory './nothere'
    at Object.fs.openSync (fs.js:432:18)
    at Object.fs.readFileSync (fs.js:289:15)
    at repl:1:9
    at REPLServer.self.eval (repl.js:110:21)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.emit (events.js:95:17)
    at Interface._onLine (readline.js:202:10)
    at Interface._line (readline.js:531:8)
    at Interface._ttyWrite (readline.js:760:14)
    at ReadStream.onkeypress (readline.js:99:10)
> e = fs.readFileSync('./emptyfile')
<Buffer >
> e.toString()
''
```